modules/snooker/src/main/ts/ephox/snooker/queries/CellOperations.ts

```diff
@@ -0,0 +1,56 @@
+import { TableSelection } from '@ephox/darwin';
+import { Element as DomElement, HTMLTableCellElement, HTMLTableElement } from '@ephox/dom-globals';
+import { Arr, Fun, Option } from '@ephox/katamari';
+import { Structs } from '@ephox/snooker';
+import { Attr, Element } from '@ephox/sugar';
+import { Selections } from '../selection/Selections';
+import { SelectionTypes } from '../selection/SelectionTypes';
+
+export interface Mergable {
+  bounds: () => Structs.Bounds;
+  cells: () => Element<DomElement>[];
+}
+
+// Return an array of the selected elements
+const selection = function (cell: Element<HTMLTableCellElement>, selections: Selections): Element[] {
+  return SelectionTypes.cata(selections.get(),
+    Fun.constant([]),
+    Fun.identity,
+    Fun.constant([ cell ])
```

Comment:
This could just be `Fun.identity` now by the looks (as `Selections` passing in the cell, not the start node), which would save having to pass in the `cell` argument.

Reply:
Tried to do this and hit type weirdness. I'll make another attempt soon.